### ISSUE TITLE
[BUG] POST polling station error 404

### ISSUE THREAD

**Issue by CristiHabliuc** [code4romania/monitorizare-vot#178]
When I attempt to save the polling station data via `POST https://mv-mobile-test.azurewebsites.net/api/v1/polling-station` I get error 404 if I previously saved the same station.

For example, if I set station to `AR 1` (and haven't set it before on my account), it saves fine, but if I then go to change station and save the same one (`AR 1`), I get this:

```
{
    "result": {
        "statusCode": 404
    },
    "id": 60,
    "exception": null,
    "status": 5,
    "isCanceled": false,
    "isCompleted": true,
    "isCompletedSuccessfully": true,
    "creationOptions": 0,
    "asyncState": null,
    "isFaulted": false
}
```

If I change the station to something else I haven't used before, it works fine.

**Comment by idormenco** [comment]
on it

**Comment by RaduCStefanescu** [comment]
fixed in #180
